runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -259,6 +259,15 @@
          */
         UncaughtExceptionHandler uncaughtExceptionHandler;
         //endregion
+
+        /** {@link Duration} before a workflow times out. */
+        @Default Duration workflowTimeout = Duration.ofMinutes(10);
```

Comment:
This isn't quite what I asked for. Workflow timeouts don't belong here.

Reply:
Ok, will pass it through the method signature.